frontend/src/components/Forms/FormLayout/index.js

```diff
@@ -0,0 +1,27 @@
+import { PropTypes } from "prop-types";
+import Heading2 from "components/Typography/Heading2";
+import Heading3 from "components/Typography/Heading3";
+
+import styles from "./index.module.css";
+
+const FormLayout = ({ children, headerText, promptText }) => {
+  return (
+    <>
+      <div className={styles.headerWrapper}>
+        <Heading2>{headerText}</Heading2>
```

Comment:
The styles for the `header` and `prompt` probably need to be changed per new design but I'll leave that up to you.

Reply:
They will but we can tackle that once we have everything up and running on this page + privacy policy page.